### PR TITLE
PROD-1969 Add app server health check run at the end of the Deploy

### PR DESCRIPTION
**Pull request by SergiiPo**
Adding action to check web version and return error if wrong version present till the end of 5 minutes period

### CODE REVIEW

File: web-checkversion/action.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: 'Check web version'
+description: 'Check web version: trying to get and compare web version during time period'
+
+inputs:
+  web-version:
+    required: true
+    description: "Version value to compare"
```
Comment (agorina) [inline web-checkversion/action.yml:7]: Nit: `Expected version` may read better.

File: web-checkversion/action.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  url-path:
+    required: true
+    description: "URL to get version"
+  json-path:
+    required: true
+    description: "JSON path to extract value with jq tool"
```
Comment (agorina) [inline web-checkversion/action.yml:13]: Nit: `JSON path to extract the expected version from the API response` ?

**Review by agorina — APPROVED** [review]
Looks good!

File: web-checkversion/action.yml (hunk 1 above)
Comment (SergiiPo) [reply web-checkversion/action.yml]: Updated

**Pull request merged**